.github/workflows/ci.yml

```diff
@@ -15,6 +15,10 @@ jobs:
         with:
           cache: yarn
       - uses: actions-rs/toolchain@v1
+        with:
```

Comment:
I tried using the `rust-toolchain` file support in this action for handling this, but couldn't get it working (in particular, this action doesn't support the [modern TOML format](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file) - it hasn't been updated for years).